addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/XiaomiHandlerFactory.java

```diff
@@ -158,6 +158,8 @@ protected ThingHandler createHandler(Thing thing) {
             return new XiaomiSensorVibrationHandler(thing);
         } else if (THING_TYPE_SENSOR_AQARA_LOCK.equals(thingTypeUID)) {
             return new XiaomiSensorLockHandler(thing);
+        } else if (THING_TYPE_BASIC.equals(thingTypeUID)) {
+            return new XiaomiDeviceBaseHandler(thing);
```

Comment:
@kaikreuzer do such semi (un)supported devices fit in the bindings, I would say yes, but just a quick check.

Reply:
Same for #2548, which is in review at the moment. I use it to control different kinds of Yeelights because the Yeelight binding has not the sufficient functionality - and I love it! It gives the advanced user some hacking delight, and the normal user more support the get to an advanced level.

Reply:
ok for me as well!